pkg/service/search.go

```diff
@@ -0,0 +1,46 @@
+package service
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/fastly"
+)
+
+// SearchCommand calls the Fastly API to describe a service.
+type SearchCommand struct {
+	common.Base
+	manifest manifest.Data
+	Input    fastly.SearchServiceInput
+}
+
+// NewSearchCommand returns a usable command registered under the parent.
+func NewSearchCommand(parent common.Registerer, globals *config.Data) *SearchCommand {
+	var c SearchCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("search", "Search for a Fastly service by name")
+	c.CmdClause.Flag("name", "Service Name").Short('n').StringVar(&c.manifest.Flag.ServiceName)
```

Comment:
This doesn't need to interact with the manifest and should instead update the input struct:
```suggestion
	c.CmdClause.Flag("name", "Service Name").Short('n').StringVar(&c.input.Name)
```